time2_test.go

```diff
@@ -0,0 +1,35 @@
+package amino
+
+import (
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestDecodeTime(t *testing.T) {
+
+	type T struct {
```

Comment:
No need to have this uppercase, would be nice to give it a more descriptive title.

Reply:
adressed